src/pages/events/dialogs/new.tsx

```diff
@@ -147,12 +158,16 @@ export const EventNewIncidentDialog: React.FC<EventNewIncidentDialogProps> = ({
   const onChangeIncidentMatch = useCallback(
     (e: React.ChangeEvent<HTMLSelectElement>) => {
       const newMatch = matches?.find((m) => m.id.toString() === e.target.value);
+
+      if (e.target.value === "-1") {
+        setMatch(null);
+        setTeam(null);
+      }
```

Comment:
Same as above but when selecting a new match instead of team

Reply:
I think this behavior is good!